pkg/forwarder/forwarder_test.go

```diff
@@ -20,15 +20,17 @@ import (
 )
 
 var (
-	monoKeysDomains = map[string][]string{
-		"datadog.foo": {"monokey"},
+	testDomain           = "http://app.datadoghq.com"
+	testVersionDomain, _ = config.AddAgentVersionToDomain(testDomain, "app")
+	monoKeysDomains      = map[string][]string{
+		testVersionDomain: {"monokey"},
 	}
 	keysPerDomains = map[string][]string{
-		"datadog.foo": {"api-key-1", "api-key-2"},
-		"datadog.bar": nil,
+		testVersionDomain: {"api-key-1", "api-key-2"},
```

Comment:
Shouldn't this be `testDomain` instead ? We use `keysPerDomains` to call `NewDefaultForwarder` so I think the url should get versioned after that.
Tests are not failing because we replace `app` by the version in 
https://github.com/DataDog/datadog-agent/blob/0f837975fea0df14c6746bc12581f022031ccaf0/pkg/config/config.go#L491